idaes/core/surrogate/pysmo_surrogate.py

```diff
@@ -328,6 +328,19 @@ def _create_model(self, pysmo_input, output_label):
     def _get_metrics(self, model):
         return {"RMSE": model.errors["MSE"] ** 0.5, "R2": model.errors["R2"]}
 
+    def get_confidence_intervals(self, model, confidence=0.95):
+        """
+        Compute confidence intervals for the regression patamaters.
+
+        Args:
+            model           : Trained PysmoPolyTrainer object
+            confidence      : Required confidence interval level, default = 0.95 (95%)
+        """
+        confint_dict = {}
+        for i in model.output_labels:
+            confint_dict[i] = model._data[i].model.confint_regression(confidence)
```

Comment:
accessing a protected member of an object (here, `_data`) is generally discouraged. Is there a public method that would provide access to this? Perhaps you should add one, if not?

Reply:
``_data`` is created in this script but has no public access. There is no need to add one since I don't want the user accessing it anyway.